certified-connectors/AgilePoint NX/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1630 @@
+                },
+                "NotificationAddressUrl": {
+                  "type": "string",
+                  "description": "Notification Address URL.",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": "Notification Address URL"
+                },
+                "Triggers": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "Trigger Key Item",
+                    "x-ms-visibility": "internal",
+                    "default": "StartProcessInstance",
+                    "description": "Trigger Key Item."
+                  },
+                  "default": [
+                    "StartProcessInstance"
+                  ],
+                  "x-ms-visibility": "internal",
+                  "description": "Triggers."
+                },
+                "TargetApplications": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "Application Filter",
+                    "default": "*",
+                    "description": "Name of the Application for Trigger."
+                  },
+                  "description": "TargetApplications."
+                }
+              },
+              "required": [
+                "NotificationAddressUrl",
+                "Triggers",
+                "Name",
+                "TargetApplications"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/Admin/CreateWebhook": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "needCallback": {
+              "type": "boolean",
+              "description": "Specifies whether to wait for the Flow to complete.",
+              "title": "Is Asynchronous",
+              "enum": [
+                "true",
+                "false"
+              ]
+            },
+            "processInstanceID": {
+              "type": "string",
+              "description": "The unique ID of a process instance.",
+              "title": "Process Instance ID"
+            },
+            "workItemID": {
+              "type": "string",
+              "description": "An unique ID that represents a task.",
+              "title": "Task ID"
+            },
+            "workObjectId": {
+              "type": "string",
+              "description": "An ID for an object, such as a document, that is associated with the process instance.",
+              "title": "Work Object ID"
+            },
+            "activityInstanceid": {
+              "type": "string",
+              "description": "The unique ID for an activity instance.",
+              "title": "Activity Instance ID"
+            }
+          }
+        },
+        "description": "AgilePartResponse"
+      },
+      "post": {
+        "responses": {

```

Comment:
@agilepointapps Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
@sriyen-msft  I have done the changes accordingly and updated